### PR TITLE
Build 3.0.0-pre.1085

### PR DESCRIPTION
**Pull request by tejal-shah**

### CODE REVIEW

File: v3/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "codap3",
-  "version": "3.0.0-mvp.1",
+  "version": "3.0.0-1085",
```
Comment (kswenson) [inline v3/package.json:3]: I think we should use a version number that makes it clear that this is not an official release -- something like `3.0.0-pre.1085` or `3.0.0-alpha.1085`.

**Review by kswenson — CHANGES_REQUESTED** [review]
👍 Generally looks good, but I think we should use a version number that makes it clear that this is not an official release -- something like `3.0.0-pre.1085` or `3.0.0-alpha.1085` as suggested below.

**Comment by tejal-shah** [comment]
@kswenson , I've updated the version number to 3.0.0-pre.1085

**Review by kswenson — APPROVED** [review]
👍 LGTM

**Pull request merged**